examples/SurfaceDial/SurfaceDial.ino

```diff
@@ -23,37 +23,43 @@ int pinB = 3;
 // input pin for pushbutton
 int pinButton = 4;
 
-volatile bool previousButtonValue = false;
+unsigned long keyPrevMillis = 0;
+const unsigned long keySampleIntervalMs = 25;
+byte longKeyPressCountMax = 840;    // 80 * 25 = 2000 ms
+byte longKeyPressCount = 0;
```

Comment:
why does this work with a count rather than with millis directly?

Reply:
idk i found the code online and adapted it for this purpose